spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/resolver/ModuleResolver.java

```diff
@@ -30,21 +30,11 @@
 	 *
 	 * @param groupId the groupId
 	 * @param artifactId the artifactId
-	 * @param version the version
-	 * @return the resource
-	 */
-	public Resource resolve(String groupId, String artifactId, String version);
-
-	/**
-	 * Retrieve a resource given its coordinates.
-	 *
-	 * @param groupId the groupId
-	 * @param artifactId the artifactId
-	 * @param version the version
-	 * @param classifer classifier
 	 * @param extension the file extension
+	 * @param classifer classifier
+	 * @param version the version
 	 * @return the resource
 	 */
-	public Resource resolve(String groupId, String artifactId, String version, String classifer, String extension);
+	public Resource resolve(String groupId, String artifactId, String extension, String classifer, String version);
```

Comment:
typo: classifer (was there before but I just noticed)... in the javadoc param as well